apps/sms/views/shared/js/inter_instance_event_dispatcher.js

```diff
@@ -13,7 +13,7 @@
   var channel = null;
 
   const BROADCAST_CHANNEL_NAME = 'messages-channel';
-  const ALLOWED_EVENTS = ['drafts-changed'];
+  const ALLOWED_EVENTS = ['drafts-saved', 'drafts-deleted'];
```

Comment:
__optional nit:__ maybe singular form will be better since now we fire these events only for one draft only ```draft-saved``` and ```draft-deleted```?